ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-release-4.10.yaml

```diff
@@ -119,6 +123,13 @@ tests:
           cpu: 100m
           memory: 200Mi
     workflow: optional-operators-ci-aws
+- as: e2e-openstack-nfv
+  optional: true
+  steps:
+    cluster_profile: openstack-nfv
+    dependencies:
+      OO_INDEX: ci-index
+    workflow: openshift-e2e-openstack-sriov
```

Comment:
If the goals if to test sriov and dpdk, isn't the nfv workflow the one allowing that?
https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift/e2e/openstack/nfv/openshift-e2e-openstack-nfv-workflow.yaml#L15-L16
the same would apply to `openshift-sriov-network-operator-release-4.9.yaml`